web/docs/guides/local-development.mdx

```diff
@@ -9,24 +9,6 @@ import TabItem from '@theme/TabItem';
 
 Supabase provides a CLI so that you can develop your application locally, rather than connecting to a live project.
 
-## Before you begin
-
-If you're using the CLI, please record your steps! You can help us improve by adding missing details and fixing errors. 
-
-Our CLI is still under heavy development, and is missing some features:
-
-- The self-hosted version of Supabase does not include a UI yet.
-We are working on this in stages, starting with our [UI library](https://github.com/supabase/ui) 
-([more context](https://github.com/supabase/supabase/discussions/1001#discussioncomment-558696)).
-
-In the meantime, here are some suggested tools for interacting with your Postgres Database:
-
-- `pgadmin`: https://www.pgadmin.org 
-- `dbeaver`: https://dbeaver.com 
-- `BeeKeeper`: https://beekeeperstudio.io
-- `HeidiSQL`: https://heidisql.com
-- `Table Plus`: https://www.tableplus.io
-
```

Comment:
@soedirgo - I think you were going to update this whole page with the CLI Tour right? Let me know, otherwise I can make some edits